src/eradiate/scenes/biosphere/_rami_scenarios.py

```diff
@@ -0,0 +1,183 @@
+    TWO_LAYER_CANOPY_PLANOPHILE_SPARSE_ERECTOPHILE_SPARSE = "HOM38_DIS_PES"
+    ADJACENT_CANOPIES_SPARSE_ERECTOPHILE_DENSE_PLANOPHILE = "HOM29_DIS_EM0"
+    ADJACENT_CANOPIES_MEDIUM_ERECTOPHILE_SPARSE_PLANOPHILE = "HOM30_DIS_ED0"
+
+
+class RAMIScenarioVersion(Enum):
+    ORIGINAL = "original"
+    SIMPLIFIED = "simplified"
+
+
+def generate_name(
+    scenario_name: typing.Union[
+        RAMIActualCanopies,
+        RAMIHeterogeneousAbstractCanopies,
+        RAMIHomogeneousAbstractCanopies,
+    ],
+    version: RAMIScenarioVersion = RAMIScenarioVersion.ORIGINAL,
+) -> str:
+    """
+    Generate a name for a scenario based on its name and version.
+
+    Parameters
+    ----------
+    scenario_name : (RAMIActualCanopies | RAMIHeterogeneousAbstractCanopies | RAMIHomogeneousAbstractCanopies)
+        The name of the scenario.
+    version : ScenarioVersion
+        The version of the scenario.
+
+    Returns
+    -------
+    str
+        The name of the scenario.
+    """
+    return (
+        f"{scenario_name.value}-{version.value}"
+        if version == RAMIScenarioVersion.SIMPLIFIED
+        else scenario_name.value
+    )
+
+
+def _convert_to_enum(
+    scenario_name: typing.Union[
+        RAMIActualCanopies,
+        RAMIHeterogeneousAbstractCanopies,
+        RAMIHomogeneousAbstractCanopies,
+        str,
+    ]
+) -> typing.Union[
+    RAMIActualCanopies,
+    RAMIHeterogeneousAbstractCanopies,
+    RAMIHomogeneousAbstractCanopies,
+]:
+    """
+    Convert a scenario name to an enum if it is a string.
+
+    Parameters
+    ----------
+    scenario_name : (RAMIActualCanopies | RAMIHeterogeneousAbstractCanopies | RAMIHomogeneousAbstractCanopies | str)
+        The name of the scenario.
+
+    Returns
+    -------
+    (RAMIActualCanopies | RAMIHeterogeneousAbstractCanopies | RAMIHomogeneousAbstractCanopies)
+        The name of the scenario as an enum.
+    """
+
+    if isinstance(scenario_name, str):
+        for member in itertools.chain.from_iterable(
+            [
+                RAMIActualCanopies,
+                RAMIHeterogeneousAbstractCanopies,
+                RAMIHomogeneousAbstractCanopies,
+            ]
+        ):
+            if scenario_name == member.value:
+                return member
+        else:
+            raise ValueError(f"Scenario {scenario_name} not found")
+    else:
+        return scenario_name
+
+
+def load_rami_scenario(
+    scenario_name: typing.Union[
+        RAMIActualCanopies,
+        RAMIHeterogeneousAbstractCanopies,
+        RAMIHomogeneousAbstractCanopies,
+        str,
+    ],
+    version: RAMIScenarioVersion = RAMIScenarioVersion.ORIGINAL,
+    padding: int = 0,
+    unpack_folder: typing.Optional[Path] = None,
+) -> dict:
+    """
+    Load a scenario based on its name and version.
+
+    Parameters
+    ----------
+    scenario_name : (RAMIActualCanopies | RAMIHeterogeneousAbstractCanopies | RAMIHomogeneousAbstractCanopies | str)
```

Comment:
IIRC, the correct numpydoc type specification for unions is
```suggestion
    scenario_name : str or RAMIActualCanopies or RAMIHeterogeneousAbstractCanopies or RAMIHomogeneousAbstractCanopies
```
(putting `str` first to make sure users won't miss it!)